OpenRA.Game/Server/Server.cs

```diff
@@ -1011,12 +1167,52 @@ public void StartGame()
 			// TODO: Enable for multiplayer (non-dedicated servers only) once the lobby UI has been created
 			LobbyInfo.GlobalSettings.GameSavesEnabled = Type != ServerType.Dedicated && LobbyInfo.NonBotClients.Count() == 1;
 
+			if (recorder != null)
+			{
+				gameInfo = new GameInformation
+				{
+					Mod = Game.ModData.Manifest.Id,
+					Version = Game.ModData.Manifest.Metadata.Version,
+					MapUid = Map.Uid,
+					MapTitle = Map.Title,
+					StartTimeUtc = DateTime.UtcNow
+				};
+				recorder.Metadata = new ReplayMetadata(gameInfo);
+				foreach (var conn in Conns)
+				{
+					var client = GetClient(conn);
+					if (client.IsObserver)
+						continue;
+					var player = new GameInformation.Player
+					{
+						ClientIndex = client.Index,
+						Name = client.Name,
+						IsHuman = client.Bot == null,
+						IsBot = client.Bot != null,
+						FactionName = "Random", // FIXME
```

Comment:
Style nit: Can we please have a single `// TODO: Populate metadata with real values` comment above this block instead of multiple contextless FIXMEs?

Reply:
Sure, I added the list on the comment though